src/locales/en-US.json

```diff
@@ -317,7 +317,7 @@
                 "invoke_address": "Smart Contract addresses can only start with KT1",
                 "not_exist": "This account doesn't exist on the blockchain or has 0 balance.",
                 "not_smartcontract": "This is a delegated address and not a smart contract. Please use send tab to transfer funds.",
-                "use_interact": "This is a smart contract address. Please use interact with contracts button to transfer funds."
+                "use_interact": "It looks like you are trying to send funds to a smart contract. Unless this smart contract explicitly supports sending, it may be impossible to withdraw your funds. To avoid losing funds, please use the \"Interact with Smart Contracts\" functionality to invoke a specific function unless you are directed by the smart contract owner."
```

Comment:
This is a bit "Clippy".

Maybe this instead?

`You are about to send funds to a smart contract. It may be impossible to recover them. To avoid unsupported uses, consider calling the contract from the \"Interact with Smart Contracts\" view, unless you were specifically instructed to use the send function by the contract owner.`

I would consider removing `unless...`

Reply:
@navi-arneja Thoughts?

Reply:
As discussed on the release meeting, we are happier with the existing language.